torch/_dynamo/bytecode_transformation.py

```diff
@@ -56,6 +56,74 @@ def create_instruction(name, arg=None, argval=_NotProvided, target=None):
         opcode=dis.opmap[name], opname=name, arg=arg, argval=argval, target=target
     )
 
+# Python 3.11 remaps
+def create_jump_absolute(target):
+    if sys.version_info >= (3, 11):
+        return create_instruction("JUMP_FORWARD", target=target)
+    return create_instruction("JUMP_ABSOLUTE", target=target)
+
+def create_pop_jump_if_true(target):
+    if sys.version_info >= (3, 11):
+        return create_instruction("POP_JUMP_FORWARD_IF_TRUE", target=target)
+    return create_instruction("POP_JUMP_IF_TRUE", target=target)
+
+def create_load_global(name, arg, push_null):
+    if sys.version_info >= (3, 11):
+        # arg = (arg << 1) + push_null
+        arg <<= 1
+    return create_instruction(
+        "LOAD_GLOBAL", arg, name
+    )
+
+def create_dup_top():
+    if sys.version_info >= (3, 11):
+        return create_instruction("COPY", 1)
+    return create_instruction("DUP_TOP")
+
+def create_rot_n(n):
+    if n <= 1:
+        return []
+    if sys.version_info >= (3, 11):
+        return [create_instruction("SWAP", i) for i in range(n, 1, -1)]
+    if n <= 4:
+        return [create_instruction(
+            "ROT_" + ["TWO", "THREE", "FOUR"][n-2]
+        )]
+    return [create_instruction("ROT_N", n)]
+
+def create_call_function(nargs, push_null=True):
+    if sys.version_info >= (3, 11):
+        output = []
+        # if push_null:
+        output.append(create_instruction("PUSH_NULL"))
+        output.extend(create_rot_n(nargs + 2))
+        output.append(create_instruction("PRECALL", nargs))
+        output.append(create_instruction("CALL", nargs))
+        return output
+    return [create_instruction("CALL_FUNCTION", nargs)]
+
+def create_call_method(nargs):
+    if sys.version_info >= (3, 11):
+        return [
+            create_instruction("PRECALL", nargs),
+            create_instruction("CALL", nargs)
+        ]
+    return [create_instruction("CALL_METHOD", nargs)]
+
+def create_pop_block():
+    if sys.version_info >= (3, 11):
+        return create_instruction("NOP")
```

Comment:
Is making this a no-op actually ok? Is that going to be a problem when handling try/except blocks?